src/lib/environment/mod.rs

```diff
@@ -371,6 +371,10 @@ fn setup_env_for_rust() -> RustInfo {
         "CARGO_MAKE_RUST_TARGET_VENDOR",
         &rustinfo.target_vendor.unwrap_or("unknown".to_string()),
     );
+    envmnt::set(
```

Comment:
i wonder if maybe just use set_optional here and not default to unknown.
as for triple, unknow is not really useful. what do you think?

Reply:
Agreed.